src/components/Scroller/Section.tsx

```diff
@@ -6,33 +6,33 @@ import { sizes } from '../../helpers';
 import { useViewportSize } from '../../hooks';
 
 type SectionProps = {
-	isActive: boolean;
+	$isActive: boolean;
```

Comment:
To avoid warning when isActive is passed on to the dom element
https://styled-components.com/docs/api#transient-props